components/trade/overviewModal/Content.vue

```diff
@@ -13,23 +13,30 @@
         v-if="trade.isAnyTokenInCollectionDesired"
         :trade="trade"
         :send-item="sendItem"
+        :title="desiredTitle"
```

Comment:
props required string, but `desiredTitle` can be `string | undefined`

Reply:
updated